python/assistant-stream/src/assistant_stream/assistant_stream_chunk.py

```
@@ -35,8 +35,13 @@ class DataChunk:
     type: str = "data"
 
+@dataclass
+class ErrorChunk:
+    error: str
+    type: str = "error"
+
 
 # Define the union type for AssistantStreamChunk
 AssistantStreamChunk = Union[
-    TextDeltaChunk, ToolCallBeginChunk, ToolCallDeltaChunk, ToolResultChunk, DataChunk
+    TextDeltaChunk, ToolCallBeginChunk, ToolCallDeltaChunk, ToolResultChunk, DataChunk, ErrorChunk
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)